From 260b2f61e9be932ac5b4f9a7ca9cc316345589fe Mon Sep 17 00:00:00 2001
From: Ashok Kumar <ashok2.kumar@amd.com>
Date: Tue, 24 May 2022 06:16:37 -0500
Subject: [PATCH 1/1] New DBus property has been added for CPU info

New DBus property CPUID1 and CPUID2 has been added in
xyz/openbmc_project/Control/Power/Cap.interface.yaml

Signed-off-by: Ashok Kumar <ashok2.kumar@amd.com>
---
 xyz/openbmc_project/Control/Power/Cap.interface.yaml | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/xyz/openbmc_project/Control/Power/Cap.interface.yaml b/xyz/openbmc_project/Control/Power/Cap.interface.yaml
index 2994b14..a293636 100644
--- a/xyz/openbmc_project/Control/Power/Cap.interface.yaml
+++ b/xyz/openbmc_project/Control/Power/Cap.interface.yaml
@@ -13,3 +13,15 @@ properties:
       description: >
           Power cap enable.  Set to true to enable the PowerCap, false
           to disable it.
+
+    - name: CPUID1
+      type: string
+      description: >
+          CPU1 ID value.
+          The unit of measurement must be hexadecimal.
+
+    - name: CPUID2
+      type: string
+      description: >
+          CPU2 ID value.
+          The unit of measurement must be hexadecimal.
--
2.25.1

